### ISSUE TITLE
Regression Error while building using Browserify

### ISSUE THREAD

**Issue by bhamblok** [devongovett/fontkit#64]
Error: tried to statically call { readFile: [Function: readFile], readFileSync: [Function: readFileSync], readdir: [Function: readdir], readdirSync: [Function: readdirSync] } as a function while parsing file: /Volumes/Development/neoScores/gustaf/node_modules/fontkit/index.js
Warning: Error running grunt-browserify. Use --force to continue.

**Comment by mrdeadsake** [comment]
I get a similar error:
```ERROR in ../~/fontkit/index.js
Module build failed: 
Error: tried to statically call { 
readFile: [Function: readFile], 
readFileSync: [Function: readFileSync], 
readdir: [Function: readdir], 
readdirSync: [Function: readdirSync] } as a function```

**Comment by liborm85** [comment]
I have the same problem. You can not successfully build https://github.com/bpampuch/pdfmake.

**Comment by devongovett** [comment]
Fixed in v1.5.2, see 733c75ef423e0ad315ba31ae20172c7d6c34308b. Apparently, brfs and babel's ES6 import implementation don't play nice together. Switched from `import` to `require`.

**Comment by mrdeadsake** [comment]
@devongovett Thanks for addressing this so quickly!
